### ISSUE TITLE
download From Youtube

### ISSUE THREAD

**Issue by Guisch** [markokajzer/discord-soundbot#29]
Hi,

Thanks for this bot, we have so much fun with friends with it.
It would be could that the bot use youtubedl to easily add songs from youtube

For example:
!addYt https://www.youtube.com/watch?v=UhKaxrH4QeY john

And it download the video, convert it into mp3 and add it as john command.

I will have a look at it, but I never touch at TypeScript. Should not be so different from JS

**Comment by markokajzer** [comment]
Hey man, thanks for the suggestion! I will look into it, but I'm currently really busted from thesis work. In the meantime, do you just download it by yourself with youtube-dl and add it manually?

Glad to hear you guys are having fun with the bot!

**Comment by Guisch** [comment]
Yeah, I actually use ```youtube-dl --extract-audio --audio-format mp3 --audio-quality 0 <url>``` then send it to the bot via discord and the !add command

**Comment by markokajzer** [comment]
I see. I agree, as you said it could be nice if the bot would just be able to do that.
How often do you find the right sounds on youtube directly? Normally, we always have to cut the vids first.

**Comment by Guisch** [comment]
Actualy, most of the time there is a short version of what I want. I'm just adding "short" tag in youtube search.
For example, for the famous video "the rap battle", adding "short" let me find this https://www.youtube.com/watch?v=Fdza03XOmTw wich is perfect for the bot

**Comment by markokajzer** [comment]
Ok, then I'll definitely consider it as a useful feature!
